### PR TITLE
enable multiline in fluent-bit-common

### PR DESCRIPTION
**Pull request by pfrcks**

### CODE REVIEW

**Comment by pfrcks** [comment]
@ganga1980 should the similar be done for geneva-logs_tenant and geneva-logs_infra fluent-bit conf files?

**Review by jatakiajanvi12 — APPROVED** [review]

File: build/common/installer/scripts/fluent-bit-conf-customizer.rb [hunk 1]
```diff
@@ -2,6 +2,7 @@
 require_relative "ConfigParseErrorLogger"
 
 @fluent_bit_conf_path = "/etc/opt/microsoft/docker-cimprov/fluent-bit.conf"
+@fluent_bit_common_conf_path = "/etc/opt/microsoft/docker-cimprov/fluent-bit-common.conf"
```
Comment (ganga1980) [inline build/common/installer/scripts/fluent-bit-conf-customizer.rb:6]: we need to do same the for fluent-bit-common.conf file in windows also.

File: build/common/installer/scripts/fluent-bit-conf-customizer.rb [hunk 2]
```diff
@@ -2,6 +2,7 @@
...
```
Comment (ganga1980) [inline build/common/installer/scripts/fluent-bit-conf-customizer.rb:5]: we need to do same the for fluent-bit-common.conf file in windows also.

Comment (pfrcks) [reply build/common/installer/scripts/fluent-bit-conf-customizer.rb:5]: @ganga1980 thanks for pointing it out. added it below.

**Review by ganga1980 — APPROVED** [review]

**Pull request merged**